Vagrantfile

```diff
@@ -57,7 +60,7 @@ if [ "up", "provision" ].include?(ARGV.first)
   install_dependent_roles
 end
 
-ANSIBLE_INVENTORY_PATH = if !ENV["VAGRANT_ENV"].nil? && ENV["VAGRANT_ENV"] == "TEST"
+ANSIBLE_INVENTORY_PATH = if testing?
```

Comment:
At the moment I doubt that will be a big deal for this project, but good to know if we need it.